build.gradle

```diff
@@ -37,6 +37,7 @@ repositories {
 // 'api' configuration allows to expose dependencies with 'compile' scope in pom
 // 'implementation' configuration allows to expose dependencies with 'runtime' scope in pom
 dependencies {
+    implementation(platform("org.springframework.boot:spring-boot-dependencies:2.6.2"))
```

Comment:
Very minor, I think convention is
```
implementation platform('org.springframework.boot:spring-boot-dependencies:2.6.2')
testImplementation 'org.springframework:spring-test'
```
(https://docs.gradle.org/current/userguide/platforms.html#sub:bom_import)

Reply:
I was wondering about this.
Just to see how it behaves, I tried the kotlin syntax which is supported as well

I can revert back to the groovy documented one.
I am just asking myself if the migration to kotlin scripts is a subject we would like to think about or not

Reply:
Not for the next weeks but yes, Kotlin script shall be a topic at some point !

Reply:
reverted in 5d3a64b